R/tutorial/client_notebook.ipynb

```diff
@@ -0,0 +1,143 @@
+{
```

Comment:
if you intend to include this notebook, let's clear the result cells.

Reply:
sounds good. Will do!